internal/backend/temporalclient/client.go

```diff
@@ -98,15 +82,15 @@ func New(cfg *Config, z *zap.Logger) (Client, error) {
 		}
 
 		if cfg.StartDevServerIfNotUp {
-			if err := impl.healthcheck(ctx); err != nil {
+			if err := impl.healthCheck(ctx); err != nil {
 				var unavailable *serviceerror.Unavailable
 				if !errors.As(err, &unavailable) {
 					return nil, fmt.Errorf("temporal client: %w", err)
 				}
 
-				z.Info("cannot connect to server, starting dev server")
+				z.Info("cannot connect to temporal, starting dev server")
```

Comment:
do you want to use `Temporal`  or `temporal`?

Reply:
Standardized "Temporal" in all log messages.